packages/ra-core/src/auth/usePermissions.ts

```diff
@@ -42,25 +34,28 @@ const emptyParams = {};
  *     };
  */
 const usePermissions = <Permissions = any, Error = any>(
-    params = emptyParams
-): State<Permissions, Error> => {
-    const [state, setState] = useSafeSetState<State<Permissions, Error>>({
-        isLoading: true,
-    });
-    const getPermissions = useGetPermissions();
-    useEffect(() => {
-        getPermissions(params)
-            .then(permissions => {
-                setState({ isLoading: false, permissions });
-            })
-            .catch(error => {
-                setState({
-                    isLoading: false,
-                    error,
-                });
-            });
-    }, [getPermissions, params, setState]);
-    return state;
+    params = emptyParams,
+    queryParams: UseQueryOptions<Permissions, Error> = {
+        refetchOnMount: false,
+    }
```

Comment:
Can you explain why you chose these defaults?

Reply:
Shouldn't you use a `staleTime` instead?

Reply:
sure, i'm fine with staleTime too.
What do you think is a sane default? 60s? 5min?

Reply:
5 mins is OK

Reply:
updated :)